packages/@lwc/state/src/__tests__/index.test.ts

```diff
@@ -0,0 +1,110 @@
+import { describe, expect, test, vi } from 'vitest';
+import { defineState } from '../index.js';
+
+const state = defineState((atom, computed, update, _fromContext) => (arg: number) => {
+  const count = atom(arg);
+  const doubleCount = computed({ count }, ({ count: countValue }) => (countValue as number) * 2);
```

Comment:
One thing I'd like to see once the lazy-compute changes are done, is to spy on `doubleCount`. That'll allow us to ensure that `doubleCount` is only invoked once after `count` is updated. There might be similar things worth doing for other parts of the state manager. For example, have multiple state atoms that feed into a single computed, and then make sure that computed is only invoked once when both atoms change.